### ISSUE TITLE
Spring Boot Dashboard throws an error while trying to perform the Start command

### ISSUE THREAD

**Issue by gabipetrovay** [microsoft/vscode-spring-boot-dashboard#121]
I get the following notification:

```
Command 'Spring Boot Dashboard: Start ...' resulted in an error (command 'spring-boot-dashboard.localapp.start-multiple' not found)
```

when I run the command (`command` + `3`):
```
>Spring Boot Dashboard: Start ...
```

I am using the latest VS Code on a MacBook OS Catalina:

```
Version: 1.53.2
Commit: 622cb03f7e070a9670c94bae1a45d78d7181fbd4
Date: 2021-02-11T11:45:54.515Z (9 hrs ago)
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Darwin x64 19.6.0
```

with the "Spring Boot Extension Pack" `v0.0.8` (containing "Spring Boot Dashboard" `0.2.0`).

I tried to:

* remove the VS Code application and installed it again, but all the preferences are kept after the new installation.
* uninstalled all the Spring Boot extension (and the extension pack) and installed them again
* downgraded "Spring Boot Dashboard" to version `0.1.10` where it was behaving the same and the upgraded back

None of these helped.

**Comment by Eskibear** [comment]
> (command 'spring-boot-dashboard.localapp.start-multiple' not found)

It's because springboot dashboard extension was not activated. It's activated when you expand the dashboard.

As I mentioned in https://github.com/microsoft/vscode-spring-boot-dashboard/issues/120#issuecomment-769581671 , the dashboard is visible only when Java extension is working on "standard mode". 

Please check status of the Java extension. When you open a .java file, check if there's a "thumb up" icon in status bar. 

![image](https://user-images.githubusercontent.com/2351748/108298531-15f22f00-71d8-11eb-9145-9162a385ae7b.png)

**Comment by mattyas-yahya** [comment]
yes, this is like my problem, i can fix this issue using plugin Spring Boot Extension Pack and using this version : 
https://marketplace.visualstudio.com/_apis/public/gallery/publishers/vmware/vsextensions/vscode-spring-boot/1.48.2023072904/vspackage

**Comment by lyJava** [comment]
> 是的，这就像我的问题，我可以使用插件 Spring Boot Extension Pack 并使用此版本解决此问题： https://marketplace.visualstudio.com/_apis/public/gallery/publishers/vmware/vsextensions/vscode-spring-boot/1.48.2023072904/vspackage

Your answer is valid, very good! Thank you! The spring boot dashboard plugin is starting to work normally now。

<img width="320" alt="Image" src="https://github.com/user-attachments/assets/bac183b0-11fa-49da-b696-c35e35f5da16" />
